index.html

```diff
@@ -9,12 +9,134 @@
     <link rel="shortcut icon" href="images/lightbeam-48.png" type="image/x-icon">
     <link rel="icon" href="images/lightbeam-48.png" type="image/x-icon">
     <link rel="stylesheet" href="css/style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
     <script src="js/store.js" type="text/javascript"></script>
     <script src="js/viz.js" type="text/javascript"></script>
   </head>
   <body>
-    <h1>Lightbeam</h1>
-    <canvas id="canvas" width="300" height="300"></canvas>
+    <aside class="controls">
+    <header>
+      <img class="logo" role="banner" src="images/lightbeam_150x45.png
+">
+    </header>
+    <nav role="navigation">
+      <h2>Visualization</h2>
+      <div class="nav__vis">
+        <button class="nav__button nav__button--vis"><img class="icon icon--button" src="images/lightbeam_icon_graph.png">Graph</button>

```

Comment:
`<button class="icon graph-icon">Graph</button>`

 looks like it could be used over the other code:
```
<button class="nav__button nav__button--vis"><img class="icon icon--button" src="images/lightbeam_icon_graph.png">Graph</button>
```